src/Normalizer/Normalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Http\Client\Plugin\Normalizer;
+
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Normalize a request or a response into a string or an array
```

Comment:
can you please also explain in the class docblock that this is `@internal` and should not be used outside this package?